Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml

```diff
@@ -9,7 +9,7 @@ configuration:
   type: 4
 - display: Admin Email
   name: admin_email
-  required: false
+  required: false  # TODO Ask if we need to change this to required
```

Comment:
If it was not required by now, not without consulting with TPM as it will become breaking change.